source/help/apps/desktop-changelog.rst

```diff
@@ -1,8 +1,119 @@
 Desktop Application Changelog
-=============================
+========================================
+
+Release v3.5.0
+--------------
+
+Release date: December 14, 2016
+
+Improvements
+~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+-  URL address is shown when hovering over links with a mouse
+-  Added CTRL+SHIFT+MINUS as a shortcut for decreasing font size
+   (zooming out)
+-  Reduce upgrade issues by properly clearing cache when updating the
+   desktop app to a new version (the application cache will be purged
+   whenever the desktop app version changes)
+-  When disconnected from Mattermost, the "Cannot connect to Mattermost"
+   page is now properly aligned at the top of the window
+-  Suppressed error messages when launching the app from the command
```

Comment:
Not sure I understand how this is an improvement, maybe re-word to make it a bit more clear?